.github/workflows/js-release.yml

```
@@ -0,0 +1,79 @@
+name: '[js] Release'
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - package.json
+
+jobs:
+  release:
+    name: Create Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: 'Setup NodeJS 20.x'
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20.x
+
+      - name: 'Get version from package.json'
+        run: |
+          VERSION=$(jq -r '.version' package.json)
+          echo "VERSION=$VERSION" >> $GITHUB_ENV
+          echo "TAG_NAME=Sails-JS/v$VERSION" >> $GITHUB_ENV
```

To be consistent with the other tags
```suggestion
          echo "TAG_NAME=js/v$VERSION" >> $GITHUB_ENV
```